### PR TITLE
Merge please

### PR DESCRIPTION
**Pull request by blavian**
basic functionality of user profile

### CODE REVIEW

File: public/stylesheets/profile.css [hunk 1]
```diff
@@ -0,0 +1,52 @@
+body{
+    background-color:#e8c766
+}
+ .navbar{
+   display: flex;
+    flex-direction: row;
+    width: 60%;
+    margin: auto;
+    align-items: center;
+background-color:#f4f1ea;
```
Comment (genagain) [inline public/stylesheets/profile.css]: Please indent this line.

File: public/stylesheets/profile.css [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+}
+.navBar-links{
+   font-family:Lato, "Helvetica Neue", Helvetica, sans-serif;
+   font-size:20px;
+   text-decoration:none;
+   padding:1rem;
+   color:blue
```
Comment (genagain) [inline public/stylesheets/profile.css]: Please add spaces after the colons for this class.

File: public/stylesheets/profile.css [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+}
+.navBar-links:hover{
+background-color: #e8c766;
```
Comment (genagain) [inline public/stylesheets/profile.css]: Please indent this line

File: public/stylesheets/profile.css [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
```
Comment (genagain) [inline public/stylesheets/profile.css]: Please remove this white space.

File: public/stylesheets/profile.css [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+}
+.home{
+   flex-grow:1
+}
+.container{
+    box-shadow: 0 1px 2px rgba(0,0,0,0.15);
+   width: 60%;
```
Comment (genagain) [inline public/stylesheets/profile.css]: Please fix the indentation here.

File: public/stylesheets/profile.css (hunk 2 above)
Comment (genagain) [reply public/stylesheets/profile.css]: Also please add a semi colon for the color property.

File: public/stylesheets/profile.css [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+   letter-spacing: normal;
+   border-bottom: 1px solid red; 
+}
+.movieImage{
+   max-width:150px;
+   float:left;
+   box-shadow: blueviolet;
+}
+label.status{
+   float:left
```
Comment (genagain) [inline public/stylesheets/profile.css]: Please make sure all of the properties that don't have semi colons have them

File: routes/users.js [hunk 7]
```diff
@@ -179,9 +180,29 @@ router.post('/logout', (req, res)=>{
...
+
+router.get(
+  "/:id(\\d+)",
+  asyncHandler(async (req, res) => {
+    const currentUser = req.session.auth.userId;
+    const users = await db.User.findByPk(currentUser,{
+      include: {
+        model: db.Movie,
+      },
+
```
Comment (genagain) [inline routes/users.js]: Please remove this white space.

File: views/profile.pug [hunk 8]
```diff
@@ -0,0 +1,39 @@
...
+    title User Profile
+    link(rel='stylesheet' href='/stylesheets/profile.css')
+  body
+    div.container
+      nav.navbar
+        a(class="navBar-links home" href="/") Marvelous Movies
+        a(class="navBar-links" href="/logout") logout
+            
+
+
```
Comment (genagain) [inline views/profile.pug]: Please remove this white space.

File: views/profile.pug [hunk 9]
```diff
@@ -0,0 +1,39 @@
...
+          h1 #{movie.title}
+          h2 #{movie.releaseDate.getFullYear()}
+          h2 #{movie.voteRating}
+          label Status
+            select(class="status")
+              option(value="movie.BlockbusterShelf.status")=movie.BlockbusterShelf.status
+              option(value="watched")='Want to watch'
+              option(value="previously watched")='Watched'
+              option(value="not interested")='Currently Watching'
+            //- h2 #{movie.description}
```
Comment (genagain) [inline views/profile.pug]: Please remove this commented out code.

File: routes/users.js [hunk 10]
```diff
@@ -179,9 +180,29 @@ router.post('/logout', (req, res)=>{
...
+  })
+);
+
+
+
+
+
+
+
+
```
Comment (genagain) [inline routes/users.js]: Please remove these extra lines.

File: views/profile.pug [hunk 11]
```diff
@@ -0,0 +1,39 @@
...
+    
+
+
```
Comment (genagain) [inline views/profile.pug]: Please remove these extra lines

**Review by genagain — CHANGES_REQUESTED** [review]

**Comment by genagain** [comment]
Overall this looks good! Once, these changes are made I'll approve it.

**Comment by blavian** [comment]
fixed those changes. Thank you

**Review by genagain — APPROVED** [review]
Looks good! Ship it 🚀

**Pull request merged**